### DISCUSSION TITLE
Exporting sub catchments

### DISCUSSION THREAD

**Discussion by Aaditya-327** [vcloarec/ReosProject#132] [General]
Is it possible to export sub-catchments from the original network? I have been selecting the sub-catchment and am trying to export it, but it only exports the whole catchment in Lekan version 2.2.1.

**Comment by Aaditya-327** [comment]
I'm sorry I just saw that feature to be added in the Issues section. I will try to check with installations from experimental exe files.

**Reply by vcloarec** [reply]
Hi,
This should export all the catchments, also  sub ones and the whole catchment. You should have all in the SHP file, maybe the big one cover others?

**Reply by vcloarec** [reply]
@Aaditya-327 
Right now, I have just added the possibility to export only the current watershed by right-click on the list or on the map. This is now available on the "very experimental" since now. And should be available on the next stable version.
